packages/@react-stately/combobox/src/useComboBoxState.ts

```diff
@@ -34,7 +34,7 @@ export interface ComboBoxState<T> extends SelectState<T> {
 }
 
 type FilterFn = (textValue: string, inputValue: string) => boolean;
-interface ComboBoxStateProps<T> extends ComboBoxProps<T> {
+interface ComboBoxStateProps<T> extends AriaComboBoxProps<T> {
```

Comment:
this doesn't quite seem right, stately doesn't want the aria props or dom props
this should stay `ComboBoxProps<T>` I believe, is there a reason that it was changed?

Reply:
I thought the prop menuTrigger was supposed to be moved to the aria specific interface --- I think I understand now that it should stay in ComboBoxProps, making the changes!